tests/cross-language/scripts/decrypt_all.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+set -e
+
+export PATH=$GOPATH/bin:$GOROOT/bin:$PATH
+
+export ASHERAH_SERVICE_NAME=service
+export ASHERAH_PRODUCT_NAME=product
+export ASHERAH_KMS_MODE=static
```

Comment:
since these are used in both scripts, we should move them to the config file to avoid duplication